install_config/configuring_gce.adoc

```diff
@@ -18,6 +18,23 @@ volumes as persistent storage] for application data. After GCE is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+[[configuring-azure-permissions]]
```

Comment:
typo, this should be `configuring-gce-permissions`. Also, I'm afraid that the correct term here is GCP - Google Cloud Platform. GCE - Google Compute Engine is only a part of the GCP, and the OCP on GCP is using more parts of the GCP, for example Google Cloud Storage, which belongs to the GCP, not to the GCE.